HACKING.md

```diff
@@ -106,6 +106,47 @@ otherwise an official image can be run with
 
 FAF should be available at ```http://localhost:8080/faf/```
 
+### Reporting into FAF
+1. Set up your server URL for libreport
+
+    Insert `URL = http://localhost:8080/faf` into `/etc/libreport/plugins/ureport.conf`
+2. Create crash
+
+    Make sure you have ABRT running and that you have autoreporting enabled (`abrt-auto-reporting enabled`)
+    Install package `will-crash` and then execute `will_abort`
```

Comment:
- Make sure you have ABRT running and that you have auto-reporting enabled (`abrt-auto-reporting enabled`)
- Install package `will-crash` and then execute `will_abort`

```markdown
- Make sure you have ABRT running and that you have auto-reporting enabled (`abrt-auto-reporting enabled`)
- Install package `will-crash` and then execute `will_abort`
```